package.json

```diff
@@ -7,20 +7,33 @@
     "lint-client-test": "eslint --ext .js,.jsx -c .eslintrc-client-test test/client",
     "lint-server": "eslint -c .eslintrc-server server",
     "lint-server-test": "eslint -c .eslintrc-server-test test/server",
-    "lint": "npm run lint-server && npm run lint-client && echo 'TODO lint server(20)/client(19) tests'",
-    "test": "echo 'TODO implement server(20)/client(19) tests'",
+    "lint": "npm run lint-server && npm run lint-client && npm run lint-client-test && echo 'TODO lint server(20)/func(21) tests'",
+    "test-frontend": "node node_modules/karma/bin/karma start karma.conf.js",
+    "test-frontend-ci": "node node_modules/karma/bin/karma start --browsers PhantomJS,Firefox karma.conf.coverage.js",
+    "test-frontend-ci-win": "node node_modules/karma/bin/karma start --browsers PhantomJS,IE karma.conf.js",
+    "test-frontend-cov": "node node_modules/karma/bin/karma start karma.conf.coverage.js",
+    "test-frontend-dev": "node node_modules/karma/bin/karma start karma.conf.dev.js",
```

Comment:
@ryan-roemer: Curious, why are these all `node node_modules/karma/bin/karma` and not just `karma` or `node_modules/.bin/karma`? Some kind of Windows or CI support?